inference-engine/tests/functional/inference_engine/ngraph_reader/gather_tests.cpp

```diff
@@ -117,7 +118,133 @@ TEST_F(NGraphReaderTests, ReadGatherNetwork) {
 </net>
 )V0G0N";
     compareIRs(model, modelV7, 16, [](Blob::Ptr& weights) {
-                auto* buffer = weights->buffer().as<int64_t*>();
-                buffer[0] = 0;
-            });
-}
\ No newline at end of file
+        auto* buffer = weights->buffer().as<int64_t*>();
+        buffer[0] = 0;
+    });
+}
+
+TEST_F(NGraphReaderTests, Read_Gather7_Network) {
+    std::string model = R"V0G0N(
+<net name="saved_model" version="10">
+	<layers>
+		<layer id="0" name="data" type="Parameter" version="opset1">
+			<data shape="2,3,4" element_type="f32"/>
+			<output>
+				<port id="0" precision="FP32">
+					<dim>2</dim>
+					<dim>3</dim>
+					<dim>4</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="1" name="indices" type="Parameter" version="opset1">
+			<data shape="8,16" element_type="i32"/>
+			<output>
+				<port id="0" precision="I32">
+					<dim>8</dim>
+					<dim>16</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="2" name="GatherV2/axis" type="Const" version="opset1">
+			<data offset="0" size="8" shape="" element_type="i64"/>
+			<output>
+				<port id="1" precision="I64"/>
+			</output>
+		</layer>
+		<layer id="3" name="Gather" type="Gather" version="opset7">
+            <data batch_dims="0"/>
```

Comment:
nit: fix indentation

Reply:
left as it was before since it's nit and since in other tes